src/services/import-build.js

```diff
@@ -21,8 +23,11 @@ const parseDecodedString = (build) => {
 };
 
 const importBuild = (encodedBuild) => {
-  if (isValidBase64(encodedBuild)) {
-    const decodedString = atob(encodedBuild);
+  const compressed = Uint8Array.from(atob(encodedBuild), (c) => c.charCodeAt(0));
+  const decompressedBuild = pako.inflate(compressed);
+
+  if (isValidBase64(decompressedBuild)) {
+    const decodedString = atob(decompressedBuild);
```

Comment:
It looks like it doesn't inflate properly:
![image](https://github.com/user-attachments/assets/1363da3e-cfc1-4058-97f1-a96953cd71a6)

Reply:
Fixed!